packages/callouts/tests/site/index.html

```diff
@@ -0,0 +1,87 @@
+		</script>
+	<![endif]-->
+</head>
+<body class="uikit-body">
+	<noscript>
+		<p role="alert">This website needs JavaScript to work properly.</p>
+	</noscript>
+
+	<a href="/">&larr; back to the module index</a>
+
+	<h1>Test: callouts</h1>
+
+	<h2>Generic callout, applied directly to the paragraph</h2>
+
+	<p class="uikit-callout">A callout.</p>
+
+	<hr>
+	<h2>Generic callout applied to a container</h2>
+
+	<p>The following container has the callout class, and holds a number of paragraphs.
+
+	<div class="uikit-callout">
+		<p>Nullam quis aliquam massa. Nunc suscipit, nisi in facilisis tempus, lectus quam ullamcorper tellus, in rutrum ipsum tortor sed nisl.</p>
+		<p>Pellentesque porttitor in turpis in ornare. Quisque posuere sapien accumsan malesuada rutrum. Nulla vitae ex lacinia, rutrum est eu, tincidunt justo. Donec et ullamcorper elit, sit amet vulputate ipsum. Nulla porttitor rutrum libero sit amet sodales. Maecenas tincidunt dapibus pretium.</p>
+		<p>Duis condimentum maximus ligula sed tincidunt.</p>
+	</div>
+
+	<hr>
+	<h2>An info variant of the generic callout</h2>
+
+	<p class="uikit-callout--info">An info callout.</p>
```

Comment:
BEM means class should be: `uikit-callout uikit-callout--info`. That way you don't need to extend the entire style just for this class. Just modify it.